dashboard/app/controllers/api/v1/pd/workshop_enrollments_controller.rb

```diff
@@ -61,14 +61,6 @@ def create
 
         if @workshop.course == COURSE_BUILD_YOUR_OWN
           enrollment.update!(enrollment_params)
-
-          # Mark attendance for all sessions in Build Your Own workshops (we are not tracking attendance in
-          # this way for Build Your Own workshops, but we want to ensure that functionality for emails,
-          # certificates, etc. remains the same).
-          @workshop.sessions.each do |session|
-            attendance = Pd::Attendance.find_restore_or_create_by! session: session, teacher: user, enrollment: enrollment
-            attendance.update! marked_by_user: user
-          end
         else
           enrollment.update!(enrollment_params.merge(school_info_attributes: school_info_params))
```

Comment:
The BYOW enrollment form doesnt collect school info right? So we can't combine these conditions?

Reply:
Yep the BYOW enrollment doesn't collect school info as of now!